lib/crawler.js

```diff
@@ -359,71 +431,173 @@ Crawler.prototype._buildHttpRequest = function _buildHTTPRequest (options) {
 	}
 };
 
-Crawler.prototype._onContent = function _onContent (error, options, response) {
+Crawler.prototype._buildHttp2Session = function _buildHttp2Session(targetHost) {
+	const self = this;
+
+	self.http2Connections[targetHost] = http2.connect(targetHost);
```

Comment:
let's assign a short variable like `currentSession` or `newSession`, so that we can make the code more readable. have a look at the code below:
```js
const newSession = self.http2Connections[targetHost] = http2.connect(targetHost);
newSession
.on('') 
.on('')
.on('')
```